rpm/ssu.spec

```diff
@@ -194,7 +194,29 @@ if [ "$1" == 0 ]; then
   getent group ssu >/dev/null && groupdel ssu
 fi
 
-%post
-/sbin/ldconfig
```

Comment:
This ldconfig removal accident?

Reply:
Yes, I thought we'd already done filetriggers for this. Reverted.